src/Exceptions/YourMembershipException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace P2A\YourMembership\Exceptions;
+
+/**
+ * Generic YourMembershipException
+ *
+ * Used typically for errors dealing with XML Parsing Issues
+ *
+ */
+class YourMembershipException extends \Exception
```

Comment:
As we discussed it's a generic Parent class so external libraries can be aware of YourMembership library exceptions